gcn_classic_to_json/notices/CALET_GBM_FLT_LC/__init__.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+from ... import utils
+
+
+def parse(bin):
+    assert bin[12] == 0, "Unused. According to docs: 'Always 0 for FLT_LC'"
+    bin[1]  # Unused. According to docs: 'Generally set to 1.'
+    bin[2]  # Unused. According to docs: 'hopcount item is defunct'.
+    bin[3]  # Unused. According to docs: 'seconds of day when packet was created'.
```

Comment:
Are these common to all notice types? If so, please move these statements to `gcn_classic_to_json/notices/__init__.py`.